tf2onnx/utils.py

```diff
@@ -437,3 +448,52 @@ def is_onnx_domain(domain):
     if domain is None or domain == "":
         return True
     return False
+
+
+def build_onnx_op(node):
+    """Build onnx op"""
+    onnx_node = helper.make_node(node.type, node.input, node.output, name=node.name)
+    # deal with attributes
+    attr = []
+    attr_graphs = node.get_body_graphs()
+    if attr_graphs:
+        for attr_name, sub_graph in attr_graphs.items():
+            copied_sub_graph = copy.deepcopy(sub_graph)
+            graph_proto = copied_sub_graph.make_graph("graph for " + node.name + " " + attr_name)
+            attr.append(helper.make_attribute(attr_name, graph_proto))
+    attr.extend([a for a in node.attr_onnx.values()])
+    if attr:
+        onnx_node.attribute.extend(attr)
+    return onnx_node
+
+
+def infer_onnx_shape_dtype(node, input_shapes, input_dtypes, opset=constants.PREFERRED_OPSET):
```

Comment:
let's make opset required.

Reply:
i am considering to move this to tf2onnx/schemas.py?
put these onnx def/schema related api calls together.